templates/appliance/openhab/_next-steps.html

```diff
@@ -1,30 +1,33 @@
-<section class="p-strip--light">
-  <div class="row u-equal-height">
-    <div class="col-6">
+<section class="p-section--deep">
+  <div class="row--50-50">
+    <hr class="p-rule" />
+    <div class="col">
       <div>
         <h2>
-          Start using your {{ appliance.name }} Ubuntu Appliance
+          Start using your
+          <br />
+          {{ appliance.name }} Ubuntu Appliance
```

Comment:
Is there a scenario where the appliance wouldn't be 'openhab'?
```suggestion
          openHAB Ubuntu Appliance
```
I think this has to always be 'openHAB'